Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:22.04 AS builder
+
+WORKDIR /go/src/github.com
+ARG TARGETARCH
+
+RUN apt update && apt install -y \
+    libcurl4-openssl-dev libssl-dev libaio-dev libnl-3-dev libnl-genl-3-dev libgflags-dev libzstd-dev libext2fs-dev libgtest-dev libtool zlib1g-dev e2fsprogs \
+    sudo pkg-config autoconf automake \
+    g++ cmake make wget git curl \
+    && apt clean \
+    && wget https://go.dev/dl/go1.22.5.linux-${TARGETARCH}.tar.gz \
+    && tar -C /usr/local -xzf go1.22.5.linux-${TARGETARCH}.tar.gz \
+    && rm go1.22.5.linux-${TARGETARCH}.tar.gz 
```

Comment:
![image](https://github.com/user-attachments/assets/2966f390-bee2-4933-8976-b8b31d8b1dbf)
Thanks！But this file has two trailing whitespaces that caused the DCO checker to fail.